### PR TITLE
added more error handling for thumbnail and retry

### PR DESCRIPTION
**Pull request by sandman45**

### CODE REVIEW

File: Radiant/fileWatcher.js [hunk 1]
```diff
@@ -152,16 +152,18 @@ const uploadFile = function (info, endStream){
...
                     const mainPath = pathFind[0].substr(0, pathFind[0].length - 1);
-
+                    const thumbnailKey = data.Key.split('-')[0];
+                    const segment = data.Key.split('-')[1];
                     if(ext === 'm3u8' && _.has(streamTracker[info.path], 'm3u8') && !streamTracker[info.path].m3u8){
                         streamTracker[info.path].m3u8 = true;
                         setTimeout(() => {
                             Logger.log(`CREATING VIDEO STREAM - conversationTopicId = ${streamTracker[info.path].conversationTopicId} fileKey = ${info.path.replace(/^.*[\\\/]/, '')} `);
-                            const thumbnailKey = data.Key.split('-')[0];
+                            // const thumbnailKey = data.Key.split('-')[0];
```
Comment (laynemoseley) [inline Radiant/fileWatcher.js]: can this be deleted?

Comment (sandman45) [reply Radiant/fileWatcher.js]: no... it NEEDS to be there.. jk ill remove it

File: Radiant/fileWatcher.js [hunk 2]
```diff
@@ -170,6 +171,14 @@ const uploadFile = function (info, endStream){
                         }, process.env.TIMEOUT_TO_CREATE_VIDEO_OBJECT);
                     }
 
+                    const seg = segment.substr(1,segment.length);
+                    const seggy = seg.split('.')[0];
```
Comment (laynemoseley) [inline Radiant/fileWatcher.js]: seg and seggy? 🧐

Comment (sandman45) [reply Radiant/fileWatcher.js]: you are just jealous of my naming conventions

Comment (laynemoseley) [reply Radiant/fileWatcher.js]: I have no idea what seggy meany

**Pull request merged**